test/system/home_test.rb

```diff
@@ -83,4 +83,12 @@ class HomeTest < ApplicationSystemTestCase
     find('.niconico-calendar-nav').assert_text 1.month.ago.strftime('%Y年%-m月')
     assert_current_path(/niconico_calendar=#{1.month.ago.strftime('%Y-%m')}/)
   end
+
+  test 'show test events on dashboard' do
+    travel_to Time.zone.local(2017, 0o4, 0o1, 10, 0, 0o0) do
```

Comment:
`0o` は8進数を使うときにつけると便利ですが、普段は使わない方が分かりやすいと思います。参考: [リテラル](https://docs.ruby-lang.org/ja/latest/doc/spec=2fliteral.html)

10進数で書く場合は `0o` をつけなければ良いです。

```suggestion
    travel_to Time.zone.local(2017, 4, 1, 10, 0, 0) do
```

Reply:
@sinsoku 
アドバイスありがとうございます！！🙇‍♂️
参考にした所と同じ形式にしたものと思います。🙇‍♂️
ご指摘頂いたところも注意して進めたいと思います！(^^

@komagata 
レビュー依頼後にすみません🙇‍♂️
こちらも変更しておりますので、併せてよろしくお願い致します🙇‍♂️